patches/tModLoader/Terraria/ModLoader/ModAccessorySlot.cs

```diff
@@ -0,0 +1,83 @@
+using Microsoft.Xna.Framework;
+using System.Collections.Generic;
+using Terraria.UI;
+
+//NOTE: Does not fully support: ItemLoader.GetGamepadInstructions and related gamepad stuff.
+
+namespace Terraria.ModLoader
+{
+	/// <summary>
+	/// A ModAccessorySlot instance represents a net new accessory slot instance. You can store fields in the ModAccessorySlot class. 
+	/// </summary>
+	public abstract class ModAccessorySlot : ModType
+	{
+		internal virtual bool skipRegister => false;
```

Comment:
needs a better name? Something about not being part of the vanilla slot list, and being manually rendered/positioned by a mod?